src/AddressRegistry.Api.BackOffice.Handlers.Lambda/AddressRegistry.Api.BackOffice.Handlers.Lambda.csproj

```diff
@@ -14,6 +14,7 @@
 
     <!-- Error NETSDK1152: https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/duplicate-files-in-output -->
     <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Why is this added?

Reply:
This is missing https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/BuildingRegistry.Api.BackOffice.Handlers.Lambda.csproj#L16

Reply:
aws-lambda-tools-defaults.json should also be updated